docs/user/using-an-OIDC-provider.md

```diff
@@ -19,11 +20,12 @@ For Kubeapps to use an Identity Provider it's necessary to configure at least th
 
 - Client ID: Client ID of the IdP.
 - Client Secret: (If configured) Secret used to validate the Client ID.
-- OIDC Issuer URL (or discovery URL): URL of the OIDC issuer.
+- Provider name (which can be oidc, in which case the OIDC Issuer URL is also required).
+- Cookie secret: a 16, 24 or 32 byte base64 encoded seed string used to encrypt sensitive data (eg. `echo "not-a-good-secret" | base64`).
 
 **Note**: Depending on the configuration of the Identity Provider more parameters may be needed.
 
-In the next sections we will explain how you can find the parameters above for some of the identity providers tested. If you have configured your cluster to use an Identity Provider you will already know some of these parameters.
+Kubeapps uses [OAuth2 Proxy](https://github.com/pusher/oauth2_proxy) to handle the OAuth2/OpenIDConnect authentication. The following sections explain how you can find the parameters above for some of the identity providers tested. If you have configured your cluster to use an Identity Provider you will already know some of these parameters. More detailed information can be found on the [OAuth2 Proxy Auth configuration page](https://pusher.github.io/oauth2_proxy/auth-configuration).
 
 ### Keycloak
```

Comment:
Are these sections still relevant? AFAIK the OIDC Issuer URL is not a needed parameter now.

Reply:
Yep, the `-oidc-issuer-url` is needed if the `oicd` provider is used - see the example I added below (it's in the additional flags, because it's not mandatory unless you're using the `oidc` provider).